lib/plugin.py

```diff
@@ -0,0 +1,103 @@
+#
+#   Copyright (c) 2019 One Identity
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to
+# deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+# sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+#
+from safeguard.sessions.plugin import AAPlugin, AAResponse
+from safeguard.sessions.plugin import HostResolver
+import re
+from fnmatch import fnmatch
+
+
+class Plugin(AAPlugin):
```

Comment:
Have you tried running with pluginv run-scenario ?
E.g.: pluginv run-scenario -n SshWithGatewayAuth

I'm guessing the plugin will ask for one time password, which we don't need so you probably have to override 
https://oneidentity.github.io/safeguard-sessions-plugin-sdk/1.1.4/aa_plugin_aa_plugin.html#safeguard.sessions.plugin.aa_plugin.AAPlugin._extract_mfa_password